docs/en_US/install/3ds.md

```diff
@@ -1,25 +1,36 @@
 # 3DS/2DS Family
 
 <div class="tip">
-	ℹ️ This guide assumes that you have a <b>Homebrewed System</b>, if you don't please follow this <a href="https://3ds.hacks.guide/" target="_blank">guide</a> on how to homebrew your system first.
+	ℹ️ This guide assumes that you have a <b>Homebrewed System running the latest version of the Luma3DS custom firmware</b>, if you don't please follow this <a href="https://3ds.hacks.guide/" target="_blank">guide</a> on how to homebrew your system first.
 </div>
 
-To connect to Pretendo Network using a 3DS/2DS system you must use the Nimbus homebrew and Luma patches
+You can get a 3DS onto Pretendo Network in 5 easy steps:
+1. [Downloading Nimbus](#downloading-nimbus-to-your-sd-card)
+2. [Enabling Luma patches](#luma-patches)
+3. [Installing Nimbus to HOME Menu](#installing-nimbus-app-to-home-menu)
+4. [Using Nimbus](#using-nimbus)
+5. [Signing into Pretendo](#signing-into-your-pnid)
 
-Navigate to the [releases](https://github.com/PretendoNetwork/Nimbus) page on the Nimbus GitHub repository
+## Downloading Nimbus to your SD card
+
+Before you start, make sure your 3DS is turned off, and insert the SD card that you use for your Nintendo 3DS into your computer.
+
+Once you've done that, navigate to the [releases](https://github.com/PretendoNetwork/Nimbus) page on the Nimbus GitHub repository.
```

Comment:
https://github.com/PretendoNetwork/Nimbus/releases/latest will always redirect to the latest release

```suggestion
Once inserted, download the latest [Nimbus release](https://github.com/PretendoNetwork/Nimbus/releases/latest).
```